main.c

```diff
@@ -686,7 +686,6 @@ static void print_formatted_result(const struct slurp_box *result,
 		}
 		printf("%c", c);
 	}
-	printf("\n");
```

Comment:
Would maybe be a good idea to `fflush(stdout)` here

Reply:
resolved in 630e1ad